docs/rst/tutorials/core/qos/persistency/persistency.rst

```diff
@@ -0,0 +1,184 @@
+
+* A database filename as the ``dds.persistence.sqlite3.filename`` property in the default participant profile.
+
+* An endpoint (``data_writer`` or ``data_reader``) profile, which is associated to the node endpoint using the
+  *topic name* as ``profile_name`` attribute.
+  In this tutorial the *topic name* will be **persistency_test**. |br|
+  The endpoint requires a `GUID <https://en.wikipedia.org/wiki/Universally_unique_identifier>`_ specified as the
+  ``dds.persistence.guid`` property. |br|
+  The endpoint Durability QoS must be set to *transient*.
+
+Environment variables set up
+----------------------------
+
+ROS 2 nodes will locate their associated xml configuration files using the ``FASTRTPS_DEFAULT_PROFILES_FILE``
+environment variable. For example:
+
+.. code-block:: bash
+
+    export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/writer_config.xml
+
+Testing Publisher Persistency
+-----------------------------
+
+The following steps will show how using persistent endpoints prevents subscribers from receiving duplicated data
+when a publisher crashes.
+
+#. Launch persistent publisher and subscriber.
+
+    .. code-block:: bash
+
+        1$ export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/writer_config.xml
+        1$ ros2 topic pub --times 5 --qos-durability system_default /persistency_test std_msgs/String "{data: 'Hello'}"
+
+        2$ export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/reader_config.xml
+        2$ ros2 topic echo --qos-durability system_default /persistency_test std_msgs/String
+
+   Note that we must specify *system_default* as durability in order to enforce the use of the xml file provided value.
+
+#. Relaunch the publisher again and check the subscriber is able to receive the samples.
+
+    .. code-block:: bash
+
+        1$ ros2 topic pub --times 5 --qos-durability system_default /persistency_test std_msgs/String "{data: 'Hello'}"
+
+#. Delete the publisher database. Now the publisher state is reset. Relaunch the publisher and check the first 10 samples are
+   discarded by the subscriber because they were already received.
+
+    .. code-block:: bash
+
+        1$ rm writer_database.db
+        1$ ros2 topic pub --times 11 --qos-durability system_default /persistency_test std_msgs/String "{data: 'Hello'}"
+
+Testing Subscriber Persistency
+------------------------------
+
+The following steps will show how using persistent endpoints prevents subscribers from receiving duplicated data
+when the subscriber crashes and it's relaunched.
+
+#. Launch persistent publisher and subscriber.
+
+    .. code-block:: bash
+
+        1$ export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/writer_config.xml
+        1$ ros2 topic pub --times 5 --qos-durability system_default /persistency_test std_msgs/String "{data: 'Hello'}"
+
+        2$ export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/reader_config.xml
+        2$ ros2 topic echo --qos-durability system_default /persistency_test std_msgs/String
```

Comment:
We have a devoted *Prerequisites* section precisely to avoid introducing *docker* commands.
Remember that using docker is optional, many people will directly their computers.

Reply:
The format, of course, I'm gonna change 😅